src/webnn_native/openvino/GraphIE.cpp

```diff
@@ -123,6 +123,45 @@ namespace webnn_native { namespace ie {
             return constantNode;
         }
 
+        IEStatusCode AddActivationNode(const ngraph_node_t* inputNode,
+                                       OperatorBase* activation,
+                                       ngraph_node_t** activationNode) {
+            IEStatusCode status = IEStatusCode::OK;
+            if (activation == nullptr) {
+                *activationNode = const_cast<ngraph_node_t*>(inputNode);
+                return status;
+            }
+            auto operatorType = activation->GetOperatorType();
+            switch (operatorType) {
+                case OperatorType::None:
```

Comment:
what does means `OperatorType::None` that will fall back to `clamp`?

Reply:
Clamp has been added in GraphBuilder, so case clamp should not add activation just like case None.

Reply:
So we don't need to define the OperatorType::None, L.131 has included the case.

Reply:
I agree, updated yet, thanks! @fujunwei